ddtrace/_trace/tracer.py

```diff
@@ -195,6 +195,24 @@ class Tracer(object):
     """
 
     SHUTDOWN_TIMEOUT = 5
+    _instance = None
+
+    def __new__(cls, *args, **kwargs):
```

Comment:
Can't we do this check in `__init__`? `__new__` is more for metaclasses

Reply:
I used __new__ becuase I can easily override this method in the Dummy, CiVis, and MLobs tracers. Overriding `__init__` would require having custom `__init__(...)` for each tracer type (ex: calling `super(DummyTracer, self).__init__()` would raise deprecation warning).

Reply:
But that would be a problem in tests only, no? I don't think we should avoid doing the right thing because of our tests 🤔

Reply:
You're right. I will fix this